CHANGELOG.md

```diff
@@ -57,7 +57,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Raised an error if the `batch_size` cannot be inferred from the current batch if it contained a string or was a custom batch object ([#10541](https://github.com/PyTorchLightning/pytorch-lightning/pull/10541))
 
 
--
+- Disable validation completely when `overfit_batches > 0` ([#9709](https://github.com/PyTorchLightning/pytorch-lightning/pull/9709))
```

Comment:
Can we also check whether docs need to be updated? They probably mention the old behaviour

Reply:
Solved

Reply:
If there is any extra info need to be stated. Left here?

Reply:
I can add extra info on logic change of `limit_test_batches` when `overfit_batches > 0`

Reply:
There are several references to the val and test dataset in trainer.rst, debugging.rst, speed.rst and maybe otheres if you do a full search for "overfit_batches"

Reply:
yeah. I'll be doing a follow-up to clean up the code a bit.